src/MatrixHttpError.ts

```diff
@@ -0,0 +1,11 @@
+export interface MatrixHttpError {
```

Comment:
One (nit): Could this be AppserviceHttpError (or MatrixAppserviceHttpError), just on the basis if not confusing it with MatrixError?

Reply:
yes!